proxy/src/http/identity.rs

```diff
@@ -296,9 +301,15 @@ mod test {
 
         let store = &*store.read().await;
         let registry = &*registry.read().await;
-        let session = session::current(peer, registry, store).await?;
+        let session = session::current(Arc::clone(&peer), registry, store).await?;
         let urn = session.identity.expect("failed to set identity").id;
 
+        // Assert that we set the default owner ans it's the same one as the session
```

Comment:
```suggestion
        // Assert that we set the default owner and it's the same one as the session
```

`:CocInstall coc-spell-checker` : P